src/test/java/seedu/task/storage/JsonAdaptedTaskTest.java

```diff
@@ -136,13 +154,13 @@ public void toModelType_deadlineAndEventOverlap_throwsIllegalValueException() {
     public void toModelType_nullAlertWindow_stillSucceed() throws Exception {
         JsonAdaptedTask task =
                 new JsonAdaptedTask(VALID_NAME, VALID_DESCRIPTION, VALID_TAGS,
-                        EMPTY_DATE, EMPTY_DATE, EMPTY_DATE, null);
+                        EMPTY_DATE, EMPTY_DATE, EMPTY_DATE, VALID_EFFORT, EMPTY_WINDOW);
         JsonAdaptedTask task1 =
                 new JsonAdaptedTask(VALID_NAME, VALID_DESCRIPTION, VALID_TAGS,
-                        EMPTY_DATE, EMPTY_DATE, EMPTY_DATE, EMPTY_WINDOW);
+                        EMPTY_DATE, EMPTY_DATE, EMPTY_DATE, VALID_EFFORT, EMPTY_WINDOW);
         JsonAdaptedTask task2 =
```

Comment:
I think we should maintain the "null" of task. The change proposed would make the assertequals duplicate.

Reply:
Thanks! Didn't notice that I changed the test case :)